kahuna/public/js/components/gu-date/gu-date.js

```diff
@@ -47,6 +47,14 @@ guDate.directive('guDate', [function () {
 
             const pika = new Pikaday({
                 field: input,
+                showTime: true,
+                showMinutes: true,
+                use24hour: true,
+                incrementHourBy: 1,
+                incrementMinuteBy: 1,
+                incrementSecondBy: 1,
+                autoClose: true,
```

Comment:
we find it jarring that the picker closes as soon as you pick a time - if I want to set a day, an hour and a minute I need to reopen the picker 3 times. Does autoClose change the behaviour so that I only need to open it once, pick all 3 things, then I can close it?

Reply:
unfortunately, `autoClose` will not change the behavior 
as mentioned [here](https://github.com/owenmead/Pikaday/issues/45)
but I tweaked the picker config, and it now keeps open after selection

Reply:
Great to see it opened! But now the little chevron that used to open and close the picker does nothing?